web/packages/plugins/shared/common/src/components/UiPluginSelector.tsx

```diff
@@ -38,16 +38,31 @@ const PathPart = styled.div`
   margin-right: 15px;
 `
 
+const getHref = (
+  dataSource: string,
+  parts: string[],
+  index: number
+): string => {
+  // Get the HREF for the document
+  return parts
+    .slice(0, index + 1)
+    .join('.')
+    .replace(`${dataSource}/`, '')
+}
+
 export function DocumentPath(props: { absoluteDottedId: string }): JSX.Element {
   const { absoluteDottedId } = props
   const parts = absoluteDottedId.split('.')
+  const dataSource = absoluteDottedId.split('/')[0]
   return (
     <PathWrapper>
       {parts.map((part: string, index: number) => {
         return (
           <div style={{ display: 'flex', flexWrap: 'nowrap' }} key={part}>
             {index !== 0 && <PathPart>/</PathPart>}
-            <PathPart>{part}</PathPart>
+            <PathPart as="a" href={getHref(dataSource, parts, index)}>
```

Comment:
can we change the color of the link in PathPart to the standard equinor green? #007079

https://eds.equinor.com/0b0c666ab/p/78c75b-colour/b/708097

Reply:
Yes, do that :)

Reply:
Fixed! Also added bold text on hover
![2022-08-02_14-34](https://user-images.githubusercontent.com/16702264/182376267-f6b891cf-c729-4b84-bd6d-16caddb560eb.png)